### PR TITLE
[MIN-88] Fix upgrading the chromadb version for streamlit application

### PR DESCRIPTION
**Pull request by dudeperf3ct**
The chromadb version in streamlit application still uses a older version `chromadb==0.3.29` and should be bumped to `0.4.3` in current [requirements.txt](https://github.com/fuzzylabs/MindGPT/blob/develop/app/requirements.txt).

Using chroma 0.4.3 also requires SQLite > 3.35. This introduced another error which can be fixed by adding `pysqlite3-binary` to requirements.txt. Additionally, following code should be added to the top of application.

```python
__import__('pysqlite3')
import sys
sys.modules['sqlite3'] = sys.modules.pop('pysqlite3')
```

Chroma issue : https://docs.trychroma.com/troubleshooting#sqlite 
Streamlit solution:  https://discuss.streamlit.io/t/issues-with-chroma-and-sqlite/47950/5

In this PR, we make the following additional changes

- [x] Upgrade chroma version and associated sqllite3 issue for streamlit application
- [x] Update seldon image to `base` instead of `small` for deployment
- [x] Fix broken paths to folder in Readme
- [x] Update `monitoring-deployment.yaml` to indicate the container image should be updated manually and add instructions for same in the Readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -107,7 +106,7 @@ To deploy a pre-trained LLM model, we first need a Kubernetes cluster with [Seld
 Apply the prepared kubernetes manifest to deploy the model:
 
 ```bash
-kubectl apply -f infrastructure/llm/seldondeployment.yaml
+kubectl apply -f infrastructure/llm_k8s/seldondeployment.yaml
```
Comment (JonoCX) [inline README.md]: `seldon-deployment.yaml`?

**Review by JonoCX — APPROVED** [review]
LGTM

File: README.md (hunk 1 above)
Comment (dudeperf3ct) [reply README.md]: I will rename the file.

**Review by osw282 — APPROVED** [review]
Well done, LGTM! 🛸

**Pull request merged**